.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "libaditof"]
 	path = libaditof
 	url = https://github.com/analogdevicesinc/libaditof.git
-	branch = main
+	branch = zmq_integration
```

Comment:
this branch will be deleted after the merge on libaditof. Please change back to main